src/Sulu/Bundle/MediaBundle/Media/FormatCache/FormatCacheClearer.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\MediaBundle\Media\FormatCache;
+
+/**
+ * Format cache clearer.
+ */
+class FormatCacheClearer
+{
+    /**
+     * @var FormatCacheInterface[]
+     */
+    protected $caches = array();
+
+    /**
+     * Clear all caches
+     */
+    public function clear()
```

Comment:
Yes,

I would register the service with an alias e.g. then the current cache when it is register as local can be cleared by.
I would say if it is called without an argument all caches would be cleared.

```
$this->caches['local']->clear()
```